### DISCUSSION TITLE
Global Sudden Death

### DISCUSSION THREAD

**Discussion by thacer13** [ppy/osu#29673] [Ideas]
This could be an addition to Sudden Death or separate mod (I like the sound of Apocalypse). It’d be globally enabled in multiplayer lobbies similar to how DT/HT is, and if any player misses a note, the map auto restarts for everybody. When everyone FC’s together, each player could have their play labeled as something like “Sudden Death: x” where x is the number of players who did it together.

This could create a great sense of team effort in lobbies, and be a nice consistency challenge.

**Comment by cl8n** [comment]
this probably doesn't make sense as a *mod* but it sounds like a fun party mode to sit alongside stuff like Tag

**Reply by thacer13** [reply]
you might be right

i was thinking of it as a mod because it would be lovely to register how many people set the FC simultaneously, somehow
